sdk/textanalytics/azure-ai-textanalytics/src/samples/java/com/azure/ai/textanalytics/TextAnalyticsAsyncClientJavaDocCodeSnippets.java

```diff
@@ -24,4 +43,892 @@ public TextAnalyticsAsyncClient createTextAnalyticsAsyncClient() {
         // END: com.azure.ai.textanalytics.TextAnalyticsAsyncClient.instantiation
         return textAnalyticsAsyncClient;
     }
+
+    // Languages
+    /**
+     * Code snippet for detecting Language
+     */
+    public void detectLanguageCodeSnippets() {
+        TextAnalyticsAsyncClient textAnalyticsAsyncClient = createTextAnalyticsAsyncClient();
+
+        // BEGIN: com.azure.ai.textanalytics.TextAnalyticsAsyncClient.detectLanguage#string
+        String inputText = "Bonjour tout le monde";
+        textAnalyticsAsyncClient.detectLanguage(inputText).subscribe(detectLanguageResult -> {
+            for (DetectedLanguage detectedLanguage : detectLanguageResult.getDetectedLanguages()) {
+                System.out.printf("Detected languages name: %s, ISO 6391 Name: %s, Score: %s.%n",
+                    detectedLanguage.getName(),
+                    detectedLanguage.getIso6391Name(),
+                    detectedLanguage.getScore());
+            }
+        });
+        // END: com.azure.ai.textanalytics.TextAnalyticsAsyncClient.detectLanguage#string
+
+        // BEGIN: com.azure.ai.textanalytics.TextAnalyticsAsyncClient.detectLanguageWithResponse#string-string
+        String input = "This text is in English";
+        String countryHint = "US";
+        textAnalyticsAsyncClient.detectLanguageWithResponse(input, countryHint).subscribe(detectLanguageResult -> {
+            for (DetectedLanguage detectedLanguage : detectLanguageResult.getValue().getDetectedLanguages()) {
+                System.out.printf("Detected languages name: %s, ISO 6391 Name: %s, Score: %s.%n",
+                    detectedLanguage.getName(),
+                    detectedLanguage.getIso6391Name(),
+                    detectedLanguage.getScore());
+            }
+        });
+        // END: com.azure.ai.textanalytics.TextAnalyticsAsyncClient.detectLanguageWithResponse#string-string
+
+        // BEGIN: com.azure.ai.textanalytics.TextAnalyticsAsyncClient.detectLanguages#List
+        final List<String> inputs = Arrays.asList(
+            "This is written in English", "Este es un document escrito en Español.");
+        textAnalyticsAsyncClient.detectLanguages(inputs).subscribe(detectedBatchResult -> {
+            System.out.printf("Model version: %s%n", detectedBatchResult.getModelVersion());
+
+            // Batch statistics
+            final TextDocumentBatchStatistics batchStatistics = detectedBatchResult.getStatistics();
+            System.out.printf("Batch statistics, document count: %s, erroneous document count: %s,"
```

Comment:
since it only shows detectLangues list, the message should need to update as well.
```suggestion
            System.out.printf("Batch statistics, document count: %s,"
```